python/fedml/api/modules/model.py

```diff
@@ -44,24 +44,28 @@ def create(name: str, model: str = None, model_config: str = None) -> bool:
 
 
 def create_from_hf(name: str, model: str = None) -> bool:
-    # Copy the template folder to the current folder
     hf_templ_fd_src = os.path.join(os.path.dirname(__file__), "..", "..", "serving", "templates", "hf_template")
-    # Copy recursively
-    dst_fd_prefix = "temp_"
-    dst_fd = os.path.join(os.getcwd(), dst_fd_prefix + name)
+    dst_parent_fd = os.path.join(os.path.expanduser("~"), ".fedml","fedml-model-client", "fedml", "hf_model_from_template")
```

Comment:
What happens if the user wants to create multiple HF model cards? Is the model card file created from template copied in another function?

Reply:
`dst_fd = os.path.join(dst_parent_fd, name)`

Reply:
I think Line 52:

Reply:
The local model card folder, says `dst_fd`, was a copy of `python/fedml/serving/templates/hf_template/` 

According to Line 52:
`dst_fd = os.path.join(dst_parent_fd, name)`
the `name` here means the model card name. So there could be multiple local model card folder with different names that created (copied) from the template folder.

Reply:
Got it. Looks good to me